inst/@sym/mrdivide.m

```diff
@@ -35,20 +34,12 @@
     return
   end
 
-
-  if isscalar(x) && isscalar(y)
-    z = rdivide(x, y);
-
-  elseif isscalar(x) && ~isscalar(y)
-    error('FIXME: scalar/array not implemented yet');
-
-  elseif ~isscalar(x) && isscalar(y)
-    z = rdivide(x, y);
-
-  else  % two array's case
-    error('FIXME: array/array not implemented yet');
+  if isscalar(x) && ~isscalar(y)
+    error("Scalar/Array not supported by this operator.")
```

Comment:
Sorry, but I still don't think Array/Array works either.

it just does `A ./ B`.  But `A \ B` should be `A * inv(B)` (but just leave not support for now I think)